content-src/asrouter/templates/SimpleBelowSearchSnippet/SimpleBelowSearchSnippet.schema.json

```diff
@@ -0,0 +1,59 @@
+{
+  "title": "SimpleBelowSearchSnippet",
+  "description": "A simple template with just an icon and rich text. It gets inserted below the Activity Stream search box.",
+  "version": "1.1.0",
+  "type": "object",
+  "definitions": {
+    "richText": {
+      "description": "Text with HTML subset allowed: i, b, u, strong, em, br",
+      "type": "string"
+    },
+    "link_url": {
+      "description": "Target for links or buttons",
+      "type": "string",
+      "format": "uri"
+    }
+  },
+  "properties": {
+    "text": {
+      "allOf": [
+        {"$ref": "#/definitions/richText"},
+        {"description": "Main body text of snippet. HTML subset allowed: i, b, u, strong, em, br"}
+      ]
+    },
+    "icon": {
+      "type": "string",
+      "description": "Snippet icon. 64x64px. SVG or PNG preferred."
+    },
+    "block_button_text": {
+      "type": "string",
+      "description": "Tooltip text used for dismiss button.",
+      "default": "Remove this"
+    },
+    "do_not_autoblock": {
```

Comment:
We probably don't need this, what do you think @glogiotatidis?

Reply:
I'd keep it for more flexibility. It defaults to `false` so it would be alright. If we need we can enable this in combination with `has_sync_enabled` to make sure that the message reaches the biggest audience it should